info.json

```diff
@@ -0,0 +1,7 @@
+"info" : {
+"name":"<project name",
```

Comment:
Missing closing &gt;

Reply:
accidental commit, removed file, the original (and fixed) is in pr #9